### ISSUE TITLE
Remove Microformat Classes from CSS

### ISSUE THREAD

**Issue by davidakennedy** [Automattic/_s#1268]
I wanted to create an issue to fix some of the issues related to errors with structured data.

We talked about these issues as a team, and adding extensive support for Microformats 2 or Schema.org through microdata or RDFa doesn't make much sense because of:

- _s being a starter theme, and it's hard to know what kind of theme will be built with it. So any support we add may be incomplete.
- Adding additional classes or attributes pollutes the markup and creates issues when people use the classes for styling.
- It's difficult because Core may/may not add additional support and help for structured data.

Going forward:

- Let's remove any Microformat-related classes from the Sass/CSS to discourage theme authors from using the classes.
- Point theme authors toward JSON-LD, which allows for adding support for structured data outside of template files or CSS. See this [page from Google on the topic](https://developers.google.com/search/docs/guides/intro-structured-data).

PRs welcome on removing all Microformat classes from the Sass/CSS files.

Related issues and PRs for reference:

#844
#860 
#930 
#1065 
#1113

**Comment by dshanske** [comment]
I disagree with this decision, but based on it, I strenuously object to endorsing JSON-LD as it is inconsistent. If you want to make this statement, it should be that _s will have no structured markup of any sort and that is to be added to any theme developed on it by the theme author.

**Comment by davidakennedy** [comment]
Thanks for speaking up, @dshanske!

> I disagree with this decision, but based on it, I strenuously object to endorsing JSON-LD as it is inconsistent.

How is it inconsistent? That would help me understand better.

> If you want to make this statement, it should be that _s will have no structured markup of any sort and that is to be added to any theme developed on it by the theme author.

Would be fine with that. I wasn't intended to add it to any official docs.

**Comment by dshanske** [comment]
Deciding that you will remove all structured markup but then recommending a different type of markup. I would rather if you are going in that direction, that you make the statement that structured markup is up to the individual who creates the theme based on _s.

**Comment by grappler** [comment]
@danhauk How is JSON-LD inconsistent? It seems like Yoast SEO thinks it is OK. https://yoast.com/json-ld/

**Comment by dshanske** [comment]
@grappler I think you meant me. But, I'm saying it is inconsistent with the decision not to support structured data. If you are rejecting one over the other, I would object.

**Comment by grappler** [comment]
> I think you meant me. But, I'm saying it is inconsistent with the decision not to support structured data.

Yes, sorry. Ah I thought you meant JSON-LD was inconsistent and not the decision to point theme authors toward JSON-LD.

> Deciding that you will remove all structured markup but then recommending a different type of markup. 

We are not removing the structured markup but removing the use of the classes from the CSS. The only change that is required is to remove `.hentry` from the CSS and replace it with `.post,.page`. I hope that clarifies it a bit.

**Comment by miklb** [comment]
Is this still planned?
